src/Sulu/Bundle/ContactBundle/Entity/AccountRepository.php

```diff
@@ -416,4 +366,16 @@ protected function appendJoins(QueryBuilder $queryBuilder, $alias, $locale)
             ->leftJoin($alias . '.categories', 'categories')
             ->leftJoin('categories.translations', 'translations');
     }
+
+    /**
+     * Create a new instance of a model.
+     *
+     * @return mixed
```

Comment:
Since you've already removed the phpdoc on all other methods I would also remove it here.